src/root/components/EmergencyThreeWForm/index.tsx

```diff
@@ -0,0 +1,931 @@
+                    label="Name"
+                    value={value?.reporting_ns_contact_name}
+                    onChange={setFieldValue}
+                    error={error?.reporting_ns_contact_name}
+                  />
+                  <TextInput
+                    name="reporting_ns_contact_role"
+                    label="Role"
+                    value={value?.reporting_ns_contact_role}
+                    onChange={setFieldValue}
+                    error={error?.reporting_ns_contact_role}
+                  />
+                  <TextInput
+                    name="reporting_ns_contact_email"
+                    label="Email"
+                    value={value?.reporting_ns_contact_email}
+                    onChange={setFieldValue}
+                    error={error?.reporting_ns_contact_email}
+                  />
+                </InputSection>
+              </>
+            )}
+            {value?.activity_lead === ACTIVITY_LEADER_ERU && (
+              <InputSection
+                title="ERU"
+                description="Which ERU is conducting the response activity?"
+              >
+                <ERUInput
+                  name={"deployed_eru" as const}
+                  value={value?.deployed_eru}
+                  onChange={setFieldValue}
+                  options={eruOptions}
+                  error={error?.deployed_eru}
+                />
+              </InputSection>
+            )}
+          </Container>
+          <Container
+            heading="Activity Reporting"
+            sub
+            visibleOverflow
+            contentClassName={styles.activityReportingContent}
+          >
+            <InputSection
+              title="Types of Actions Taken"
+              description="Select the actions that are being across all of the locations tagged above"
+            >
+              <Checklist
+                name="sectors"
+                options={sectorOptions}
+                onChange={handleSectorChange}
+                value={sectorValue}
+                keySelector={numericValueSelector}
+                labelSelector={labelSelector}
+              />
+            </InputSection>
+            {value?.sectors?.map((s, i) => {
+              if (isDefined(s.sector)) {
+                return (
+                  <SectorInput
+                    key={s.sector}
+                    index={i}
+                    sectorTitle={sectorIdToLabelMap?.[s.sector] ?? ''}
+                    actionOptions={activityOptionListBySector[s.sector]}
+                    onChange={setSector}
+                    value={s}
+                    supplyOptionListByActivity={supplyOptionListByActivity}
+                    error={getErrorObject(error?.sectors)}
+                    averageHouseholdSizeForSelectedCountry={averageHouseholdSizeForSelectedCountry}
+                  />
+                );
+              }
+
+              return null;
+            })}
```

Comment:
Use of List or ListView component is recommended.
Also, if necessary handle empty list case.